action.yml

```
@@ -13,6 +13,10 @@ inputs:
     description: 'Path to container files, including Dockerfile'
     required: false
     default: '.'
+  name:
+    description: 'Name of image'
+    required: false
+    default: 'temp'
```

```suggestion
    default: ''
```

A more sensible default would be the empty string because we can check for that down below. As unlikely as it is, maybe someone's image name is `temp`!